Refactor/filesystem

docsobral opened the pull request:
dirname.ts and fetch.ts have been merged, and a function has been moved from save.ts to the new filesystem.ts

orchidb0y commented:
A principal mudança nesse branch foi a migração do cloudinary para o supabase. Com o supabase, passamos a hospedar tanto imagens quanto os arquivos mjml e html. Cada projeto tem um bucket próprio no nome do projeto.

No ato de processamento, é criada uma pasta temporária para ambos enquanto são processados, mas logo em seguida essa pasta com arquivos é deletada. O mjml permanece no servidor em sua forma original, e o HTML processado e preparado para envio é hospedado lá. O HTML tem links temporários (10 minutos por padrão).

docsobral committed:
New module: filesystem.ts
1. moved checkFirstUse from save.ts to filesystem.ts
2. moved functions from fetch.ts to filesystem.ts
3. renamed dirname.ts to filesystem.ts

docsobral committed:
New module: filesystem.ts
1. moved checkFirstUse from save.ts to filesystem.ts
2. moved functions from fetch.ts to filesystem.ts
3. renamed dirname.ts to filesystem.ts

docsobral committed:
Merge branch 'refactor/filesystem' of https://github.com/orchidb0y/mailer into refactor/filesystem
